aspnetcore/web-api/Microsoft.dotnet-openapi

```diff
@@ -0,0 +1,118 @@
+---
+title: Develop ASP.NET Core apps using OpenAPI
+author: ryanbrandenburg
+description: Demonstrates how to use the 'Microsoft.dotnet-openapi' tool to add references to OpenAPI files.
+ms.author: rybrande
+ms.date: 08/26/2019
+monikerRange: '>= aspnetcore-3.0'
+uid: web-api/Microsoft.dotnet-openapi
+---
+# Develop ASP.NET Core apps using OpenAPI tools
+
+By Ryan Brandenburg
+
+`Microsoft.dotnet-openapi` is a .NET Core global tool for managing [OpenAPI](https://github.com/OAI/OpenAPI-Specification) references within a project.
+
+## Installation
+
+To install `Microsoft.dotnet-openapi`, run the following command:
+
+``` console
```

Comment:
This installation command is completely valid, assuming that the doc publishes next week after 3.0 ships. A `--version` option has to be provided if publishing before then.

```suggestion
```console
```

Reply:
Disregard. I see you're targeting the 3.0-GA branch, so this won't publish until after the release.